torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -619,6 +637,7 @@ struct GraphFuser {
       auto* old_output = chunk->outputs().at(i);
       auto* new_output = bchunk->outputs().at(i);
       new_output->copyMetadata(old_output);
+      aliasDb_->replaceWithNewValue(old_output, new_output);
```

Comment:
I'm not sure this is correct. what does prim::BroadcastChunk do exactly?

Reply:
Basically:
```
  // replace a,b = chunk(op(x,y,z)) with:
  // x', y', z' = broadcast_tensors([x, y, z])
  // x0,x1 = chunk(x') (x0 has a's type, x1 has b's type)
  // y0,y1 = chunk(y') (y0 has a's type, y1 has b's type)
  // z0,z1 = chunk(z') (z0 has a's type, z1 has b's type)
  // a_out = op(x0,y0,z0) (a,b have their same size but are now contiguous)
  // b_out = op(x1,y1,x1)
```
So this code is rewiring `a_out` to point to `a` and such.